src/abseil-cpp/randen_hwaes_nohw.cc

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "absl/random/internal/randen_slow.h"
+
+#include "absl/random/internal/randen_hwaes.h"
+
+namespace absl {
+ABSL_NAMESPACE_BEGIN
+namespace random_internal {
+
+void RandenHwAes::Generate(const void* keys, void* state_void) {
+  return RandenSlow::Generate(keys, state_void);
```

Comment:
If absl already provides RandenSlow():
- why can't we use it?
- isn't there already a complilation flag for absl that allows you to fallback to using RandenSlow automatically?

Reply:
RandenSlow is an implementation detail then not exposed so it's not possible for us to use it directly. Abseil automatically choose either fast or slow one by detecting whether the fast one is feasible at compile time or runtime.

There is no flag to control this. That's what I want to suggest to the Abseil team to explicitly disable it.